### ISSUE TITLE
Question about max length of sentence

### ISSUE THREAD

**Issue by xixy** [yubochen/NBTNGMA4ED#5]
Thank you very much for your open-sourced code. I noticed that you set the max length of sentence to 40 by cutting sentences longer than 40 as follows:

https://github.com/yubochen/NBTNGMA4ED/blob/e5d54455485c72dfa90a8735e31e42b25b669f54/data_utils.py#L200

Is the comparison fair considering many sentences are longer than 40? For example, the longest sentence in train/dev/test contains 228/154/84 tokens respectively. The triggers are also dropped and not counted when calculating F1. 

Thank you and look forward to your reply.

**Comment by HangYang-NLP** [comment]
The parameter of "max length of sentences" is a hyper-parameter, it can be set to 40 or others for batch training. In evaluation, it can be set as the max length of sentences on the development set.

**Comment by xixy** [comment]
Thanks for reply. Another question is did you use the max length for evaluation, which corresponds to the performance reported in your paper?

**Comment by HangYang-NLP** [comment]
Yes, we set the hyper-parameter as the max sentence length on the test set for evaluation.

**Comment by xixy** [comment]
Thanks for reply.
